### ISSUE TITLE
don't count @keyword internal : 80% documented man pages 

### ISSUE THREAD

**Issue by LiNk-NY** [Bioconductor/BiocCheck#58]
IMO, documentation using the `@keywords internal` tag should be excluded from the denominator for the error: 
```
* ERROR: At least 80% of man pages documenting exported
objects must have runnable examples.
```

**Comment by LiNk-NY** [comment]
The exception here is that there was an `@export` tag in the function.
Not sure how we should make the distinction between internal (documented) 
and exported functions or are these conflicting labels?
Exported functions should always have runnable examples I think :thinking:

**Comment by lshep** [comment]
yes exported functions should have runnable examples with few exceptions.  That was the original intention of the check I believe

**Comment by nturaga** [comment]
#59 is related.

**Comment by LiNk-NY** [comment]
It looks like this is resolved in #90 
https://github.com/Bioconductor/BiocCheck/blob/c0d2faa6b5fac1c812bfa3e5d45ea09b322254b0/R/checks.R#L1310-L1320
